sorc/ncep_post.fd/MPI_FIRST.f

```diff
@@ -85,8 +87,8 @@ SUBROUTINE MPI_FIRST()
       use ctlblk_mod, only: me, num_procs, jm, jsta, jend, jsta_m, jsta_m2,           &
               jend_m, jend_m2, iup, idn, icnt, im, idsp, jsta_2l, jend_2u,            &
               jvend_2u, lm, lp1, jsta_2l, jend_2u, nsoil, nbin_du, nbin_ss,           &
-              nbin_bc, nbin_oc, nbin_su
-
+              nbin_bc, nbin_oc, nbin_su,                                              & 
+              isx,iex
```

Comment:
@GeorgeVandenberghe-NOAA Would you rename isx, iex as ista, iend which are consistent with jest and jend for j decomposition?

Reply:
ista and iend are already being used in para_range.f.  The conflict can be resolved because they aren't used outside
but I will have to use other names inside this code